api/migrations/versions/0edb89e87e72_indabaxza_offer_email_templates.py

```diff
@@ -0,0 +1,187 @@
+        application_close,
+        review_open,
+        review_close,
+        selection_open,
+        selection_close,
+        offer_open,
+        offer_close,
+        registration_open,
+        registration_close,
+        event_type,
+        travel_grant,
+        miniconf_url=None
+    ):
+        self.start_date = start_date
+        self.end_date = None if event_type == EventType.CONTINUOUS_JOURNAL else end_date
+        self.key = key
+        self.organisation_id = organisation_id
+        self.email_from = email_from
+        self.url = url
+        self.application_open = application_open
+        self.application_close = application_close
+        self.review_open = review_open
+        self.review_close = review_close
+        self.selection_open = selection_open
+        self.selection_close = selection_close
+        self.offer_open = offer_open
+        self.offer_close = offer_close
+        self.registration_open = registration_open
+        self.registration_close = registration_close
+        self.event_roles = []
+        self.event_type = event_type
+        self.travel_grant = travel_grant
+        self.miniconf_url = miniconf_url
+        self.event_fees = []
+
+class EmailTemplate(Base):
+
+    __tablename__ = 'email_template'
+    __table_args__ = tuple([db.UniqueConstraint('key', 'event_id', 'language', name='uq_email_template_key_event_id')])
+
+    id = db.Column(db.Integer(), primary_key=True)
+    key = db.Column(db.String(50), nullable=False)
+    event_id = db.Column(db.Integer(), db.ForeignKey('event.id'), nullable=True)
+    language = db.Column(db.String(2), nullable=False)
+    template = db.Column(db.String(), nullable=False)
+    subject = db.Column(db.String(), nullable=False)
+
+    event = db.relationship('Event', foreign_keys=[event_id])
+
+    def __init__(self, key, event_id, subject, template, language):
+        self.key = key
+        self.event_id = event_id
+        self.subject = subject
+        self.template = template
+        self.language = language
+
+
+def upgrade():
+    Base.metadata.bind = op.get_bind()
+    session = orm.Session(bind=Base.metadata.bind)
+    event = session.query(Event).filter_by(key='indabax2023').first()
+
+    offer_award_content = """Dear {title} {firstname} {lastname},
+
+Congratulations! You've been selected for {event_name}!
+
+In addition, you have been awarded a travel grant to attend the IndabaX!
+
+Congratulations, only the very best have been awarded this. Your travel grant has been made possible by our partners at the event, so give them a personal thanks and chat about future opportunities.
+{grants}
+
+The terms and conditions are specified on our website and may be subject to change - https://indabax.co.za/register/travel-grants
+
+Please note that breakfast and lunches are provided but dinner is not, although evening events may have snacks available.
+
+Please follow this link to see details and accept your offer: {host}/offer
```

Comment:
Please change the link to 
{host}/indabax2023/offer

Reply:
Right of course, so obvious now 😄 Pushed the updates (and the other 3 urls too)